src/config.spec.ts

```diff
@@ -0,0 +1,64 @@
+import fs from "fs";
+import { Config } from "./types";
+import { getDynalitePort, setConfigDir } from "./config";
+
+const BASE_PORT = 8443;
+const DEFAULT_BASE_PORT = 8000;
+
+const mockedConfig = jest.fn((): Config => ({ basePort: BASE_PORT }));
+const configPath = "/fakepath/jest-dynalite-config.js";
+
+jest.mock("fs");
```

Comment:
I actually prefer to mock with `memfs`

For example
```
jest.mock("fs", () => jest.requireActual('memfs'))
```

Then you can freely write a file with `fs.writeFile` and it won't be written to disk :)